run_analysis.sh

```diff
@@ -41,16 +51,23 @@ EMBEDDING=1                 # options: 0, 1
 
 # Build workspace
 STXS_FIT="inclusive"        # options: stxs_stage0, stxs_stage1p1, inclusive
-./datacards/produce_workspace.sh $ERA $STXS_FIT | tee ${ERA}_produce_workspace_${STXS_FIT}.log
+./datacards/produce_workspace.sh $ERA $STXS_FIT $METHOD | tee ${ERA}_produce_workspace_${STXS_FIT}.log
 
 # Run statistical inference
 #./combine/significance.sh $ERA | tee ${ERA}_significance.log
-./combine/signal_strength.sh $ERA $STXS_FIT | tee ${ERA}_signal_strength_${STXS_FIT}.log
-./combine/signal_strength.sh $ERA "robustHesse" | tee ${ERA}_signal_strength_robustHesse.log
-./combine/diff_nuisances.sh $ERA
+./combine/signal_strength.sh $ERA $STXS_FIT $DATACARDDIR/cmb/125 cmb ${METHOD}
+# ./combine/signal_strength.sh $ERA "robustHesse" $DATACARDDIR/cmb/125 cmb ${METHOD}
+# ./combine/diff_nuisances.sh $ERA
 #./combine/nuisance_impacts.sh $ERA
 
 # Make prefit and postfit shapes
-./combine/prefit_postfit_shapes.sh $ERA
-./plotting/plot_shapes.sh $ERA $STXS_SIGNALS $CATEGORIES $JETFAKES $EMBEDDING $CHANNELS
+
+# ./combine/prefit_postfit_shapes.sh ${ERA} ${STXS_FIT} ${DATACARDDIR}/cmb/125 ${METHOD}
+
+# PREFITFILE="${DATACARDDIR}/prefitshape-${era}-${METHOD}-${STXS_FIT}.root"
```

Comment:
This should be ${DATACARDDIR}/cmb/125/prefitshape-${ERA}-${METHOD}-${STXS_FIT}.root